code/__DEFINES/species/species.dm

```diff
@@ -7,6 +7,8 @@
 #define SPECIES_ID_PROMETHEAN "promethean"
 #define SPECIES_ID_TESHARI "teshari"
 #define SPECIES_ID_VOX "vox"
+#define SPECIES_ID_SHADEKIN "shadekin"
```

Comment:
why define an additional ID, shadekin and crewkin are already distinct as species names

Reply:
During talk with Zandario, I was told that we might as well define both of them with the new Species_ID System. This way, both Shadekin and Black-Eyed Shadekin have the same Species ID, while Black-Eyed Shadekin have the sub-species ID of Black-Eyed Shadekin.
Might be useful in the future if we have things that are supposed to work differently with Shadekin, it'll effect Black-Eyed ones as well.